src/Microsoft.AspNet.Mvc.Formatters.Json/Internal/JsonBufferPool.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNet.Mvc.Formatters.Json.Internal
+{
+    public class JsonBufferPool<T> : IJsonBufferPool<T>
```

Comment:
Is there a benefit to making it generic as opposed to it being `IJsonBufferPool<char>`?